backend/src/main/java/gov/cdc/usds/simplereport/config/GraphQlConfig.java

```diff
@@ -61,6 +62,11 @@ public DataFetcherExceptionResolver dataFetcherExceptionResolver() {
             singletonList(new GenericGraphqlException(exception.getMessage(), errorPath)));
       }
 
+      if (exception instanceof TestEventSerializationFailureException) {
+        return Mono.just(
+            singletonList(new GenericGraphqlException("Error saving result", errorPath)));
```

Comment:
⚠️ Let me know if this message should change 🤔

Reply:
question: is there a reason we've made a new exception with this custom error message rather than following the `exception.getMessage()` pattern as above? Wondering when we should use that way of grabbing the error message and when we should make a new one like you have here

Reply:
I think in this case, the actual `TestEventSerializationFailureException` doesn't have a pretty exception message or stack trace - instead it's logging the UUID of the TestEvent. That's being done so we can trace the exceptions in Azure and track down the specific test events that cause a page, but obviously isn't what we want to return to the end user. 

More generally, I actually prefer the pattern of setting the user-facing error message here, and using the exceptions themselves to track useful information about the issue. It's nice to have all the user-facing errors in one place so we can update them easily without tracking down all the places an error might be thrown, but really helpful to have more than just "failed to serialize test event" when you're trying to debug an issue.

Reply:
I'll reach out to laura with some more context and a screenshot! :) Also, thanks for your question, Bob and answering Emma 😂

Reply:
Error message looks like this now
<img width="1909" alt="Screen Shot 2022-11-20 at 09 19 07" src="https://user-images.githubusercontent.com/20211771/203150182-5b424564-af6e-4c75-9762-505c8f0229ca.png">